### ISSUE TITLE
Bug: Justify on blank line

### ISSUE THREAD

**Issue by rizen** [mimshwright/pixi-tagged-text#171]
If you have a string like "foo\n\nbar", which creates a blank line in the middle then you'll get a nasty error message in PTT and it won't render.

Uncaught (in promise) TypeError: Reduce of empty array with no initial value
    at Array.reduce (<anonymous>)
    at eval (pixi-tagged-text.m.js?5fbe:1)
    at n (pixi-tagged-text.m.js?5fbe:1)
    at eval (pixi-tagged-text.m.js?5fbe:1)
    at eval (pixi-tagged-text.m.js?5fbe:1)
    at eval (pixi-tagged-text.m.js?5fbe:1)
    at C.B.update (pixi-tagged-text.m.js?5fbe:1)

**Comment by mimshwright** [comment]
Example to reproduce the issue:

```
const basicText = `foo\n\nbar`;
const basicStyle = {
  default: {
    align: "justify",
  },
};
new TaggedText(basicText, basicStyle);
```

Note it only happens when align = justify

**Comment by mimshwright** [comment]
Fixed in v2.3.5
